### DISCUSSION TITLE
do we have keebio/iris/rev7 ?

### DISCUSSION THREAD

**Discussion by huydinhle** [manna-harbour/miryoku#200] [Q&A] [answered]
I saw the qmk instruction only has `keebio/iris/rev4`. Can we flash this on rev7 by any chances?

**Comment by manna-harbour** [comment, marked as answer]
Yes, that's just an (old) example. All versions are supported in the same way. 

Here's a sample build run: https://github.com/manna-harbour/miryoku_qmk/actions/runs/4420157096

**Reply by Cdavi23** [reply]
I'm a total noob at this, but is there any reason I would be getting a firmware is too large error after following this for my rev7?

**Reply by manna-harbour** [reply]
Yes, using AVR-based MCUs with QMK is becoming difficult due to firmware size. You'd be better off with e.g. ARM-based MCUs. See https://github.com/qmk/qmk_firmware/blob/master/docs/squeezing_avr.md. For the above build, I disabled RGB with `RGB_MATRIX_ENABLE=no`. See https://github.com/manna-harbour/miryoku_qmk/actions/runs/4420157096/jobs/7749502919#step:2:13
